modules/UI/videolayout/RemoteVideo.js

```diff
@@ -519,18 +518,13 @@ RemoteVideo.prototype.setDisplayName = function(displayName, key) {
             if (displaynameSpan.text() !== displayName)
                 displaynameSpan.text(displayName);
         }
-        else if (key && key.length > 0) {
```

Comment:
I think @yanas was removing this while she was removing user status changed event handling, and this parameter is no longer used.